packages/dotnet-sdk/src/TeamsFx/Configuration/AuthenticationOption.cs

```diff
@@ -27,9 +27,37 @@ public class AuthenticationOptions
     [Required(ErrorMessage = "Client secret is required")]
     public string ClientSecret { get; set; }
 
+    /// <summary>
+    /// AAD tenant id.
+    /// </summary>
+    [RegularExpression(@"^[0-9A-Fa-f\-]{36}$")]
+    public string TenantId { get; set; }
+
+    /// <summary>
+    /// Application ID URI.
+    /// </summary>
+    public string ApplicationIdUri { get; set; }
+
     /// <summary>
     /// Authority URL that is used in OAuth On-behalf-of flow.
     /// </summary>
     [RegularExpression(@"^http(s)?://[-a-zA-Z0-9@:%._\+~#=/]{1,100}$")]
     public string OAuthAuthority { get; set; }
+
+    /// <summary>
+    /// Bot specific authentication configurations.
+    /// </summary>
+    public BotAuthenticationConfigurations Bot { get; set; }
 }
+
+/// <summary>
+/// Bot specific authentication configurations.
+/// </summary>
+public class BotAuthenticationConfigurations
+{
+    /// <summary>
+    /// Initiate login start page endpoint.
+    /// </summary>
+    [Required(ErrorMessage = "login start page endpoint uri is required")]
+    public string LoginStartPageEndpoint { get; set; }
```

Comment:
in nodejs sdk, we call it initiate login endpoint, make the config name align?

Reply:
It comes from a previous discussion. I have forwarded the discussion to you.

Reply:
Fixed.